src/api/app/views/models/_user.xml.builder

```diff
@@ -13,6 +13,8 @@ xml.person do
     xml.globalrole(role.title)
   end
 
+  xml.ignore_auth_services(my_model.ignore_auth_services)
```

Comment:
Is it necessary that we expose that in the XML?

Reply:
Why should we hide it? I wouldn't see this as data we need to hide. Though we could only show it in LDAP mode though. Or only for admins.

Reply:
Not that we need to hide it but for the majority of users this information will be useless. I would limit it to admins!